apps/google-analytics-4/frontend/src/components/main-app/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  ChartData,
+  ChartOptions,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import tokens from '@contentful/f36-tokens';
+
+const ACCESSIBILITY_LABEL = 'Analytics line chart';
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip);
+
+ChartJS.defaults.font.size = 12;
+ChartJS.defaults.font.family = tokens.fontStackPrimary;
+ChartJS.defaults.font.weight = '500';
+ChartJS.defaults.borderColor = tokens.gray200;
+ChartJS.defaults.datasets.line.borderColor = tokens.colorPrimary;
+
+interface LineChartProps {
+  dataValues: number[];
+  xAxisLabels: string[];
+  tooltipMetricLabel: string;
+  accessibilityLabel: string;
+}
+
+const LineChart = ({
+  dataValues,
+  xAxisLabels,
+  tooltipMetricLabel,
+  accessibilityLabel,
+}: LineChartProps) => {
+  const data: ChartData<'line'> = {
+    labels: xAxisLabels,
+    datasets: [
+      {
+        data: dataValues,
+      },
+    ],
+  };
+
+  const options: ChartOptions<'line'> = {
+    responsive: true,
+    plugins: {
+      tooltip: {
+        backgroundColor: tokens.gray900,
+        bodyColor: tokens.colorWhite,
+        padding: 8,
```

Comment:
use `tokens.spacingXs`. it may be useful to create a function that parses the `px` off for unit-less integer formatting

Reply:
Great idea! Have created a helper function to parse px and rem off the token values.